server/views/pages/checkTransfer.njk

```diff
@@ -0,0 +1,33 @@
+{% extends "../partials/layout.njk" %}
+{% from  "../partials/checkThisRecordMacro.njk" import checkThisRecordMacro %}
+{% set pageTitle = "Check transfer" %}
+
+
+{% block content %}
+    <div class="govuk-width-container">
+        <main class="govuk-main-wrapper">
+          <span class="govuk-caption-xl"> Confirm arrival of {{ data.firstName}} {{ data.lastName }} </span>
+            <h1 class="govuk-heading-l govuk-!-margin-bottom-7">Check this record matches the person in reception</h1>     
+            <div>
+                {{ checkThisRecordMacro(data) }}
```

Comment:
Don't think this matches the design.. https://dsdmoj.atlassian.net/browse/DCS-1280